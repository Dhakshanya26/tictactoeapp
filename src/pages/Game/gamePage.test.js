import React from "react";
import GamePage from "./gamePage";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
Enzyme.configure({ adapter: new Adapter() });
describe("GamePage", () => {
  test("Check whether game header and card displayed", () => {
    const component = shallow(<GamePage />);
    expect(component).toMatchSnapshot();
  });
});
