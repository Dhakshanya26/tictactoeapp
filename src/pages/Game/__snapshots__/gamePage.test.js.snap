// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GamePage Check whether game header and card displayed 1`] = `ShallowWrapper {}`;
